docs/upgrade_db_zonemaster_backend_ver_5.0.2.md

```diff
@@ -1,4 +1,4 @@
-If your zonemaster database was created by a Zonemaster-Backend version smaller than
+IF your zonemaster database was created by a Zonemaster-Backend version smaller than
```

Comment:
What is wrong with "If"?

Reply:
Nothing, I shouldn't have touched this file. I removed this update.